Partition (WBP)/Functions/VI0049D.VTI0049B.fun.yml

```diff
@@ -0,0 +1,135 @@
+﻿Function:
+  Enabled For RDMLX:  Yes
+
+  Source: |
+    <-
+    
+    Function Options(*DIRECT) Rcv_List(#wLKData #wLRet)
+    
+    * *********************************************************************************************
+    * Test Case Description
+    * Issue#49 Continued : PerformanceMetrics data Table generation.
+    * VTI0049B will be calling Function VI0049D which the pipeline will execute after VTI0049.
+    * The test setup will reformat 'PerformanceMetricsLog.txt' into 'PerformanceMetricsTable.xls'.
```

Comment:
`PerformanceMetricsTable.txt`
Its not an XLS file. Its a comma-delimited TEXT file which Excel can render easily

Reply:
I have fixed this and pushed